server/src/services/validation/worker/build/buildInputsToSolc.ts

```diff
@@ -101,6 +102,10 @@ export async function buildInputsToSolc(
 
   getValidationFile(workerState, buildJob);
 
+  if (buildJob.context.file === undefined) {
```

Comment:
Is it 1 to 1 that all build jobs without a context file are outside the hh proj?

Reply:
This condition happens when the file is neither on the source paths or a dependency of one of them